js/profile-w3c-common.js

```diff
@@ -70,6 +70,7 @@ define(
     "core/data-tests",
     "core/list-sorter",
     "w3c/jsonld",
+    "w3c/jsonld",
```

Comment:
Oops, looks like jsonld is loaded twice

Reply:
tried to solve a merge conflict, went wrong :P
will fix on next review